python/pecos/simulators/custatevec/bindings.py

```diff
@@ -28,6 +36,7 @@
     "RY": one_q.RY,
     "RZ": one_q.RZ,
     "R1XY": one_q.R1XY,
+    "RXY1Q": one_q.R1XY,
     "SX": one_q.SX,
     "SXdg": one_q.SXdg,

```

Comment:
A few more gates that we should define...  

SqrtX == SX
SqrtXd == SXdg
SqrtYd == SY
SqrtYd == SYdg
SqrtZd == SZ
SqrtZd == SZdg

Seen these in circuits in the wild, so seems like we need them.

Reply:
Sure thing, I've added these in. 

However, rather than adding these synonyms manually to each individual simulator, I think we should make some choice of canonical names (e.g. those from the [PHIR specs](https://github.com/CQCL/phir/blob/main/spec.md#table-ii---quantum-operations)) and add a "centralised translator" in `parent_sim_classes.py` (or similar) that all commands go through before reaching the backend. I'm happy to create a separate PR to add this in, if it'd be helpful.

Reply:
I think the problem is that we have made multiple decisions on the canonical names over time and every shift creates a compatibility problem when interfacing with the package.  Being a bit more exhaustive temporarily and calling out which would be deprecated seems like the right balancing act for ease of forward movement?

Reply:
Yeah, I think being more exhaustive right now is fine. If it isn't already, very soon, everything can conform to the PHIR spec naming. (There might be some older style names like SqrtX left over in internal software that need to be fixed, and then we can clean up PECOS.)